src/index.scss

```diff
@@ -165,3 +165,57 @@ header {
 .filter-bar:focus-visible, .filter-bar:focus {
   outline: none;
 }
+
+header .user-dropdown button {
+  height: 44px !important;
+  width: 74.8167px;
+  font-size: 18px;
+  font-weight: 400;
+  border-radius: 6px ;
+  border: 1px solid #07223c;
+  line-height: 28px;
+}
+
+header .user-dropdown span {
+  font-weight: 400;
+}
+
+.learning-header {
+  margin-left: -1.25rem;
+  margin-right: -1.25rem;
+  padding-top: 1 rem;
+  line-height: 1;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  font-size: 18px;
+  height: 60px;
+}
+
+.learning-header .container-xl span {
+  line-height: 1;
+}
+
+.learning-header .container-xl .logo {
+  margin-top: 4px;
+}
+
+.learning-header span:first-child {
+  font-size: 87.5%;
+  margin-top: 2px !important;
+  margin-bottom: -2px !important;
+}
```

Comment:
You can group classes in SCSS. 

.learning-header {
  margin-left: -1.25rem;
  margin-right: -1.25rem;
  padding-top: 1 rem;
  line-height: 1;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
  font-size: 18px;
  height: 60px;

  .container-xl {
    span {
      line-height: 1;
    }

    .logo {
      margin-top: 4px;
    }
  }

  span:first-child {
    font-size: 87.5%;
    margin-top: 2px !important;
    margin-bottom: -2px !important;
  }
}
`